src/legacy/core_plugins/timelion/index.js

```diff
@@ -73,6 +73,12 @@ export default function (kibana) {
           }
         };
       },
+      savedObjectSchemas: {
```

Comment:
Can you add a comment explaining why this type is not exportable? It's not immediately obvious for this one (the telemetry ones are more obvious).